### ISSUE TITLE
Error in _check_updates_ready

### ISSUE THREAD

**Issue by jwoehr** [spyder-ide/spyder#21051]
### What steps will reproduce the problem?

Just loading Spyder and scrolling around. Didn't even load a new file. After about 30 seconds this error popped up.

### Traceback
```python-traceback
Traceback (most recent call last):
  File "/home/jwoehr/spyder_venv/lib/python3.10/site-packages/spyder/plugins/application/container.py", line 331, in _check_updates_ready
    msg = header + content + footer
UnboundLocalError: local variable 'content' referenced before assignment
```

## Versions

* Spyder version: 5.4.2  (pip)
* Python version: 3.10.6 64-bit
* Qt version: 5.15.2
* PyQt5 version: 5.15.6
* Operating System: Linux 6.2.6-76060206-generic

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.1.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.31.1;<9.0.0       :  7.34.0 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.9.0 (OK)
jsonschema >=3.2.0            :  4.5.1 (OK)
keyring >=17.0.0              :  23.5.1 (OK)
nbconvert >=4.0               :  6.5.0 (OK)
numpydoc >=0.6.0              :  1.3.1 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.1 (OK)
pygments >=2.0                :  2.12.0 (OK)
pylint >=2.5.0;<3.0           :  2.13.9 (OK)
pylint_venv >=2.1.1           :  2.3.0 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.7.1;<1.8.0          :  1.7.1 (OK)
pylsp_black >=1.2.0           :  1.2.1 (OK)
qdarkstyle >=3.0.2;<3.1.0     :  3.0.3 (OK)
qstylizer >=0.2.2             :  0.2.2 (OK)
qtawesome >=1.2.1             :  1.2.1 (OK)
qtconsole >=5.4.0;<5.5.0      :  5.4.0 (OK)
qtpy >=2.1.0                  :  2.1.0 (OK)
rtree >=0.9.7                 :  1.0.0 (OK)
setuptools >=49.6.0           :  59.6.0 (OK)
sphinx >=0.6.6                :  4.5.0 (OK)
spyder_kernels >=2.4.2;<2.5.0 :  2.4.2 (OK)
textdistance >=4.2.0          :  4.2.2 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.8 (OK)
xdg >=0.26                    :  0.27 (OK)
zmq >=22.1.0                  :  23.0.0 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=3.0.0            :  None (NOK)
numpy >=1.7                   :  1.22.4 (OK)
pandas >=1.1.1                :  None (NOK)
scipy >=0.17.0                :  None (NOK)
sympy >=0.7.3                 :  None (NOK)
```

**Comment by dalthviz** [comment]
Hi @jwoehr thank you for the feedback! This is a duplicate of #20406 To prevent this you will need to either deactivate the check for updates at startup option or update Spyder to the latest release (5.4.3). The fix for this issue was released with Spyder 5.4.3 so hopefully if you update to 5.4.3 then, when we release a new version, you will not get this error again.

Let us know if the info above helps!

**Comment by jwoehr** [comment]
Thank you @dalthviz ... I did not realize I was not up to the latest revision. I have just updated my installation.
